README.md

```diff
@@ -1,7 +1,40 @@
-# nulecule-library
-A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your applications
+# Nulecule library
 
-###Communication channels
+A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your deployment.
+
+Intended to be used with [atomicapp](https://github.com/projectatomic/atomicapp). We have a thorough [start guide](https://github.com/projectatomic/atomicapp/blob/master/docs/start_guide.md) on how to get started.
+
+### Library format
+
+__Files:__
+
+Please only include the `/artifact` folder, Nulecule, Dockerfile and a README.md.
```

Comment:
Please only include the `/artifact` directory and the Nulecule, Dockerfile and README.md files in your nulecule.